src/docs/basics.html

```diff
@@ -1,12 +1,4 @@
 ---
 layout: default
+navigationIdx: 0
```

Comment:
ID of the element from the yml config file (0 = basics, 1=components, 2=containers)